docs/troubleshooting-guide/my-dashboard-is-slow.md

```diff
@@ -4,133 +4,74 @@ title: My dashboard is slow
-2. The next steps depend on what those other applications are, how frequently they are making queries, whether the database can be replicated or the load can be moved onto other systems, and so on. If some of the applications involved are primarily being used for batch processing (e.g., daily or weekly reports), you can also check when those jobs are scheduled to ensure that they don't overlap.
+However, a standalone embedded dashboard (a [signed embed](../embedding/signed-embedding.md)) won't log people into Metabase. This means that people viewing the signed embed won't be subject to the permissions and data restrictions set up by your admin. When someone loads a question or a dashboard in a signed embed, that question or dashboard will utilize a larger, unrestricted dataset rather than a smaller, permissions-limited dataset.
 
-Note: you may also see your database being overloaded if you're using the same database for Metabase's app database and for your own data. We strongly recommend that you don't do this in a production system or if you have more than a handful of users. If you are doing this, set up a second database for Metabase's own use.
+## Dashboard is slow compared to similar dashboards
 
-## Are some of your queries intrinsically slow?
+1. Remove fields (columns) that you don't need in the final result.
+2. Add a [filter](../questions/query-builder/introduction.md#filtering) to reduce the amount of data being queried. For example:
+   - Narrow down the time frame to the reporting period that you care about.
+   - Exclude invalid records, such as: blanks, nulls, or rows with values like "cancelled", "expired", "invalid", and so on.
+3. Remove [joins](../questions/query-builder/introduction.md#joining-data) to tables that aren't being used. 
+4. If you're aggregating data from the query builder, ask your database admin if there's a pre-aggregated [view](https://www.metabase.com/glossary/view) that you can use instead.
 
-**Root cause:** Joining half a dozen tables, each with a few million rows, simply takes a lot of time. While we do our best to create fully-formed SQL queries from graphical questions, SQL snippets, and questions that use other questions as starting points, it's a hard problem---particularly across as many databases as we support.
+**Explanation**
 
-**Steps to take:**
+When you update your question to use a minimal number of rows or columns (or switch your question to use a smaller table, such as a summary table) your database can spend less time scanning those records in order to return your results. This is especially important to think about if you're [starting from someone else's saved question or model](../questions/query-builder/introduction.md#play-around-with-saved-questions), because you might not need all of the data that the original creator decided to include.
```

Comment:
```suggestion
When you update your question to use a minimal number of rows or columns (or switch your question to use a smaller table, such as a summary table) your database can spend less time scanning those records in order to return your results. Narrowing the scope of your question is especially important to think about if you're [starting from someone else's saved question or model](../questions/query-builder/introduction.md#play-around-with-saved-questions), because you might not need all of the data that the original creator decided to include.
```